volatility/framework/layers/crash.py

```diff
@@ -146,13 +152,15 @@ def _load_segments(self) -> None:
                 offset += x.PageCount
 
         elif self.dump_type == 0x05:
-            summary_header.BufferLong.count = (summary_header.BitmapSize + 31) // 32 + 0x2000
+            #Add 0x2000 as some bitmaps are too short by one offset
+            summary_header.BufferLong.count = (summary_header.BitmapSize + 31) // 32 + 0x2000 
```

Comment:
I'm still a little confused why the BufferLong calculation isn't right and more how adding 0x2000 is the right solution?  If they're short by a single offset, then shouldn't it be something like `(summary_header.BitmapSize + 32) // 32` but then the question is why is the wrong value included in the calculation?  This should essentially be calculating `math.ceil(BitmapSize / 32)`, so is that value inaccurate, or is something else going on?

Reply:
Yeah, I'm not entirely sure why the calculation is wrong either, but it is definitely off by that amount and the issue doesn't seem to be anywhere in crash.py. I'm not sure where this calculation is actually done be but if you know I could take a look